frontend/src/UserAPI.js

```diff
@@ -1,17 +1,31 @@
 import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter'
+
+var mock = new MockAdapter(axios);
 
 const api = axios.create({
     baseURL: `https://jsonplaceholder.typicode.com/`,
 })
 
+const api2 = axios.create({
+    baseURL: `http://localhost:8080/api/users/`,
+})
+
 function getUsers() {
     return api
         .get(`/users`);
 }
 
+function login(username, password) {
+mock.onPost().reply(200);
```

Comment:
A mockot mozgassuk ki top level, nem kell minden hívás előtt beállítani, elég egyszer.

Reply:
Feljebb raktam, így is jól működik. Így jó lesz?

Reply:
Igen.